crates/snops-agent/src/rpc/control.rs

```diff
@@ -624,4 +626,14 @@ impl AgentService for AgentRpcServer {
             .await
             .map_err(|_| AgentError::FailedToMakeRequest)?
     }
+
+    async fn get_status(self, ctx: context::Context) -> Result<AgentStatus, AgentError> {
+        let lock = self.state.node_client.lock().await;
+        let node_client = lock.as_ref().ok_or(AgentError::NodeClientNotSet)?;
+
+        Ok(AgentStatus {
+            aot_online: node_client.status(ctx).await.is_ok(),
```

Comment:
wouldn't the presence of the node_client lock indicate that the node is online? I guess this indicates it's responsive

Reply:
true, I could rename it to online and responsive or something? Idk I hate naming things

Reply:
online is fine